package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "single-spa-layout",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Please revert this, the changes to this are done during publish

```suggestion
  "version": "2.1.0",
```